packages/webapp/webpack/webpack.common.js

```diff
@@ -47,6 +47,10 @@ module.exports = {
         test: /\.css$/,
         use: ['style-loader', 'css-loader'],
       },
+      {
+        test: /\.svg$/,
```

Comment:
Is that really necessary? 

why don't we just include the svg as the src of a normal <img> tag. I think svgr/webpack is just an unnessecary additional dependency

Reply:
or including the svgs via the < use >

Reply:
Please see my comment below.

Reply:
We would then not be able to style it via an external CSS file, which is needed for theme-switcher icon.